recipe/install.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+if [ "${PKG_NAME}" == "librdkit" ]; then
+
+    # NOTE(skearnes): List is from `make list_install_components`, excluding "python" and "pgsql".
+    for COMPONENT in Unspecified base data dev docs extras runtime; do
+      cmake -D CMAKE_INSTALL_COMPONENT="${COMPONENT}" -P cmake_install.cmake
+    done
+
+elif [ "${PKG_NAME}" == "rdkit" ]; then
+
+    cmake -D CMAKE_INSTALL_COMPONENT=python -P cmake_install.cmake
+    cmake --build . --config Release --target stubs
+
+    # NOTE(hadim): Below we run `pip install ...` in order to correctly add the `.dist-info` directory in the package
+    # so python can correctly detect whether rdkit is installed when it's coming from a conda package.
+
+    # Set the version for setuptools_scm.
+    export SETUPTOOLS_SCM_PRETEND_VERSION="${PKG_VERSION}"
+
+    # Install the Python library.
+    ${PYTHON} -m pip install --no-deps -vv --no-build-isolation --prefix "${PREFIX}" .
```

Comment:
I think we need more pip flags here. It's trying to find the index for some reason: 

```
1 location(s) to search for versions of pip:
* https://pypi.org/simple/pip/
```

These are the environment variables that are exported by conda-build for the top-level output, but not in the sub-outputs. This will be fixed in the 24.7 release, but for now we still need them:

```
PIP_NO_INDEX=True PIP_NO_DEPENDENCIES=True PIP_NO_BUILD_ISOLATION=False PIP_IGNORE_INSTALLED=True PYTHONDONTWRITEBYTECODE=True
```

So I think it'd be safer to add `--no-index` and `--ignore-installed`. And for good measure, export `PYTHONDONTWRITEBYTECODE=1` too.

Reply:
Done